subt/main.py

```diff
@@ -1197,8 +1197,11 @@ def main():
         cfg['robot']['modules']['app']['init']['wall_dist'] = args.wall_dist
         if args.side == 'auto':
             cfg['robot']['modules']['app']['init']['right_wall'] = 'auto'
+            cfg['robot']['modules']['app']['init']['robot_name'] = f'{args.timeout}C'
```

Comment:
As discussed yesterday, this should have a one-character prefix.

I am also not sure if 'auto' and 'C' are the same thing or not.

Reply:
it is not and both actually breaks Virtual (I guess). `auto` was meant that it will come from external source (via channel `robot_name` now.

Reply:
p.s. I miss in `subt/main.py` change from skiddy, i.e. use virtual if you have robot_name??

Reply:
Being virtual or not is/can also be configured in the json file. I agree that setting `virtual_world: true` for a physical world is strange. Ideally, we would not even have the non-virtual code-path(s). What do you think?